lib/routes/thoughtworks/index.ts

```diff
@@ -0,0 +1,75 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { parseDate } from '@/utils/parse-date';
+
+const bearerToken = 'Bearer xxf6ea8ed2-9d54-4d48-b5e2-b4119f83dd6d'; // 固定的 token
```

Comment:
The token can be updated from the server-side. It's suggested to visit `https://www.thoughtworks.com/rest/search/config` and use the value of `BLOG_SEARCH_TOKEN` instead of a fixed string.